src/libcmd/command.cc

```diff
@@ -91,6 +91,12 @@ EvalCommand::EvalCommand()
         .description = "start an interactive environment if evaluation fails",
         .handler = {&startReplOnEvalErrors, true},
     });
+
+    addFlag({
+        .longName = "ignore-try",
+        .description = "ignore exceptions in try clauses during debug",
+        .handler = {&ignoreExceptionsDuringTry, true},
+    });
```

Comment:
Maybe better to make this a setting inside `EvalSettings`. That way it can be set in `nix.conf`.